app/backend/src/clj/teet/map/map_services.clj

```diff
@@ -112,12 +112,26 @@
        (handle-wfs-response wfs-url (:TYPENAME query-params)
                             request-delay (or custom-parse-feature parse-feature))))))
 
-(defn wfs-get-feature [{:keys [wfs-url]} typename ogc-filter]
+(defn- propertyName->string
+  "Transforms a vec of keyword property names into a comma separated list:
+   [:a :b :c] -> \"a,b,c\""
+  [propertyName]
+  (->> propertyName
+       (map name)
+       (str/join ",")))
```

Comment:
This now allows one to restrict the properties fethed from WFS. For example, the road geometries are irrelevant for the task at hand.